docker/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM debian:8.2
```

Comment:
I'm not too docker literate, but might this be easily implemented off of a `scratch` base to reduce size? If it's too hard to get the binary in the container with `scratch` it doesn't need to be changed (although couldn't we just build before a docker push?).